### ISSUE TITLE
[IMP] Etherpad Template

### ISSUE THREAD

**Issue by lasley** [clouder-community/clouder#196]
I was looking through the Odoo template, and I noticed that there's no Etherpad. In Odoo v9 & 10 there is the ability to have some awesome collab features in the Project module if you have an Etherpad server.

I've used [Etherpad Lite](https://github.com/ether/etherpad-lite) with success. Any thoughts on alternatives?

**Comment by YannickB** [comment]
Nope I agree, Etherpad is a must have.

There is also the suggestion of onlyoffice #149 but IMO both are complementary.

**Comment by lasley** [comment]
Agreed they're complimentary - Odoo needs something using the Etherpad API protocol, which I don't believe OnlyOffice supports. Realistically Etherpad probably isn't even worth exposing to anything other than Odoo instance.
